### PR TITLE
Issue 62 improvements clarifications

### PR DESCRIPTION
**Pull request by loostrum**
I've addressed all points raised in issue #62. Comments welcome of course.

### CODE REVIEW

File: _episodes/02-benchmarking.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ The `%%timeit` line magic does exactly this in a concise an comfortable manner!
 `%%timeit` first measures how long it takes to run a command one time, then
 repeats it enough times to get an average run-time. Also, `%%timeit` can measure run times without
 the time it takes to setup a problem, measuring only the performance of the code in the cell.
-This way we can trust the outcome better.
+This way we can trust the outcome better. Instead of timing an entire cell,
+it is also possible to time a single line with `%time` or `%timeit`.
```
Comment (jhidding) [inline _episodes/02-benchmarking.md:69]: I don't want to raise any gnarly questions about line magics. Most instructors will know this and can make a point out of it if they desire, but I think it distracts from what we want to show.

File: _episodes/02-benchmarking.md [hunk 2]
```diff
@@ -145,57 +146,36 @@ visualize([rprof2], output_notebook())
...
-x = [timeit.timeit(
-        stmt=f"da.arange(5*10**7).sum().compute(num_workers={n})",
-        setup="import dask.array as da",
-        number=1)
-     for n in range(1, 9)]
+x = []
+for n in range(1, 9):
+    time_taken = %timeit -r 1 -o da.arange(5*10**7).sum().compute(num_workers=n)
+    x.append(time_taken.average)
 ~~~
```
Comment (jhidding) [inline _episodes/02-benchmarking.md:179]: Ah, now I see the reason... I didn't know you could use line magics this way.

File: _episodes/03-computing-pi.md [hunk 3]
```diff
@@ -206,6 +206,33 @@ function into machine code just before it is called (that's what the `@numba.jit
...
+> {: .source}
+> ~~~
+> CPU times: user 58.3 ms, sys: 3.27 ms, total: 61.6 ms
+> Wall time: 60.9 ms
+> CPU times: user 5 µs, sys: 0 ns, total: 5 µs
+> Wall time: 7.87 µs
+> ~~~
+> {: .output}
+{: .callout}
+
```
Comment (jhidding) [inline _episodes/03-computing-pi.md:235]: awesome!

**Comment by jhidding** [comment]
Hi Leon, I like the changes. If you add the suggested example of getting the result of a line-magic into a variable I can merge.

File: _episodes/02-benchmarking.md (hunk 1 above)
Comment (loostrum) [reply _episodes/02-benchmarking.md:69]: Good point, I'll remove my addition

**Comment by loostrum** [comment]
Great, I've made the suggested changes.

**Pull request merged**